packages/block-library/src/latest-comments/index.php

```diff
@@ -26,7 +26,7 @@
 	 * @param int|WP_Post $post Optional. Post ID or WP_Post object. Default is global $post.
 	 * @return string The post title if set; "(no title)" if no title is set.
 	 */
-	function gutenberg_draft_or_post_title( $post = 0 ) {
+	function wp_latest_comments_draft_or_post_title( $post = 0 ) {
```

Comment:
This is a handy little utility function we could use in lots of places in core.

Any chances we could make this more generic, e.g. just `wp_draft_or_post_title()`? We could add this to WordPress core and remove it from Gutenberg even.

cc @pento

Reply:
@swissspidy according to the comment, that function should already be in core, but now looking for it I can't find it in `wp-includes/template.php` or other spots.

Reply:
These functions are all in `src/wp-includes/blocks/latest-comments.php`.

Reply:
That being said, `wp_latest_comments_draft_or_post_title()` could totally be renamed and moved to another, more generic file.